php/generate_descriptor_protos.sh

```diff
@@ -13,17 +19,16 @@ __EOF__
   exit 1
 fi
 
-pushd src
-./protoc --php_out=internal:../php/src google/protobuf/descriptor.proto
-./protoc --php_out=internal_generate_c_wkt:../php/src \
-  google/protobuf/any.proto \
-  google/protobuf/api.proto \
-  google/protobuf/duration.proto \
-  google/protobuf/empty.proto \
-  google/protobuf/field_mask.proto \
-  google/protobuf/source_context.proto \
-  google/protobuf/struct.proto \
-  google/protobuf/type.proto \
-  google/protobuf/timestamp.proto \
-  google/protobuf/wrappers.proto
+$PROTOC --php_out=internal:php/src google/protobuf/descriptor.proto
```

Comment:
Yep, definitely!  This is down as one of the follow-up tasks.  PHP was largely neglected in this PR simply because it was recently reworked significantly and wasn't tied up in the whole autotools/tests.sh/kokoro mess.